ml-agents/mlagents/trainers/tests/torch/test_simple_rl.py

```diff
@@ -37,11 +42,19 @@
 
 PPO_TORCH_CONFIG = ppo_dummy_config()
 SAC_TORCH_CONFIG = sac_dummy_config()
+COMA_TORCH_CONFIG = coma_dummy_config()
 
 # tests in this file won't be tested on GPU machine
 pytestmark = pytest.mark.check_environment_trains
 
 
+@pytest.mark.parametrize("action_sizes", [(0, 1), (1, 0)])
```

Comment:
Can this be tested for combinations of rank 1, 2 and 3 observations and with and LSTM config? (To make sure it does not crash at least)

Reply:
Added some tests for LSTM, variable length obs, and visual